genanki/note.py

```diff
@@ -49,11 +49,12 @@ def insert(self, i, tag):
 class Note:
   _INVALID_HTML_TAG_RE = re.compile(r'<(?!/?[a-z0-9]+(?: .*|/?)>)(?:.|\n)*?>')
 
-  def __init__(self, model=None, fields=None, sort_field=None, tags=None, guid=None):
+  def __init__(self, model=None, fields=None, sort_field=None, tags=None, guid=None, due=None):
```

Comment:
You can achieve almost exactly the same behavior by changing this to `due=0` and removing the `@property` and its setter, so I think it'd be good to simplify it.